scout/server/blueprints/alignviewers/views.py

```diff
@@ -62,7 +60,15 @@ def remote_cors(remote_url):
 @alignviewers_bp.route("/remote/static", methods=["OPTIONS", "GET"])
 def remote_static():
     """Stream *large* static files with special requirements."""
-    file_path = request.args.get("file") or ""
+    file_path = request.args.get("file") or "."
```

Comment:
Would it be really difficult to just replace that file arg with case-individual-filetype/configkey, much as in #3252? That way, the the actual path could be kept internal to the app instead of passing via the users browser. I can have a look if you are tired of this issue by now. 😊

Reply:
It's quite a lot of code because it's not just case key/value. I see some issues a

- Sometimes the track is at the case level, sometimes at the individual level
- The tracks have an index that should also be passed by this function, and the index file suffix is extrapolated by the main track suffix 
- The tracks need to be built anyway before (in the controllers) because already in the template you need to specify which type of track igv should expect (wig, alignment, bed, etc. see [here](https://github.com/Clinical-Genomics/scout/blob/1200f4a0f947f35e1fbae8f1403d4e59ca0f8fd2/scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html#L120)) , so you'd end up having almost the same code twice

Reply:
But it if you want to try, please do! Perhaps in a separate branch, because I'll instroduce the code to purge the session in this one

Reply:
Yeah, I think 1 and 2 above should be rather possible. It's harder to get around 3 without adding some complexity. One could perhaps have a common helper for the two, producing the same track info and use just the file name for this one, but I haven't tried so no idea yet if there is anything that is difficult to get around. 

How about we focus on merging this, which a lot safer than before, and we can refactor later as time allows?

Reply:
yes, I agree!